//// [tests/cases/compiler/catch.ts] ////

=== catch.ts ===
function f() {
>f : () => void
>  : ^^^^^^^^^^

    try {} catch(e) { }
>e : unknown
>  : ^^^^^^^

    try {} catch(e) { }
>e : unknown
>  : ^^^^^^^
}

